docs/FAQ.md

```diff
@@ -305,6 +305,20 @@ directory.
 
 We hope to integrate with Gerrit natively in the future.
 
+### I want to write a tool which integrates with Jujutsu, should I use the library or parse the CLI?
```

Comment:
nit: replace the comma by a period since the second part is a full sentence (or maybe i'm just old-fashioned)

Reply:
done